docs/kubernetes/pouch_with_kubernetes_deploying.md

```diff
@@ -17,7 +17,7 @@ Updated: 2018.3.30
 
 ## Overview
 
-This document shows how to easily install a Kubernetes cluster with Pouch as the container runtime.
+This document shows how to easily install a Kubernetes cluster with Pouch as the container runtime.If it's not convenience for you,there is an easy guide for you to quickly experience this amazing combination in China [Kubernetes + Pouch + Domestic](pouch_with_kubernetes_deploying_domestic.md).
```

Comment:
Hmmm...., in China.... it's wired...

You know China is an open country :)

It might be better to say:

`For Chinese customer, it's might be faster and easier to download Kubernetes  components and various kinds of container images from Aliyun ....`